config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
> how about sticking the example test in one of the other core exercises

Yep, I think only exercise with example tests currently is `clock`, so it could even be odd to have this big explanation of example tests and then not have any. The example tests in `clock` were added because the exercise was confusing, maybe for now we could add the note to that exercise and then move it to an earlier core exercise if we add more example tests?

> introducing stubs in the first core exercise after Hello World?

Yep, I think `two-fer` should be the second core exercise, and then we can rejig the next few, make a few of them bonus exercises. I can open a PR for that tomorrow.